infra/prod/_modules/app_services/app_service_fims_oidc_provider.tf

```diff
@@ -0,0 +1,34 @@
+module "appservice_fims" {
+  source = "github.com/pagopa/terraform-azurerm-v3//app_service?ref=v7.67.1"
+
+  plan_type = "internal"
+  plan_name = "${var.product}-oidc-provider-plan"
+  sku_name  = "B3"
+
+  name                = "${var.product}-oidc-provider-app"
```

Comment:
I was wrong! the right name is `openid-provider` ([reference](https://openid.net/specs/openid-connect-core-1_0.html#Terminology:~:text=an%20OpenID%20Provider.-,OpenID%20Provider%20(OP),-OAuth%202.0%20Authorization))